src/s3.js

```diff
@@ -42,16 +42,16 @@ export async function* listObjects({bucket, prefix}) {
  * Generate a signed URL for an S3 object.
  *
  * @param {object} params
- * @param {string} params.method - HEAD, GET, PUT, DELETE
  * @param {string} params.bucket - name of bucket
  * @param {string} params.key - object key
+ * @param {string} [params.method] - HEAD, GET, PUT, DELETE. Default: GET
  * @param {object} [params.headers] - Content-Type and/or Content-Encoding headers for PUT
  * @param {number} [params.expiresIn] - seconds until the URL expires
  * @param {number} [params.issuedAt] - absolute time in seconds since the Unix epoch at which the signed URL should be considered issued at, i.e. when the countdown for expiresIn starts
  * @returns {string} signed URL
  */
-export async function signedUrl({method, bucket, key, headers, expiresIn, issuedAt} = {method: "GET", headers: {}}) {
-  const normalizedHeaders = normalizeHeaders(headers);
+export async function signedUrl({method, bucket, key, headers, expiresIn, issuedAt}) {
```

Comment:
Ahh, thanks for the catch! I get this backwards sometimes. I meant to write:

```diff
-export async function signedUrl({method, bucket, key, headers, expiresIn, issuedAt} = {method: "GET", headers: {}}) {
+export async function signedUrl({method = "GET", bucket, key, headers = {}, expiresIn, issuedAt} = {}) {
```

which makes the `headers || {}` and `method || 'GET'` in the function body unnecessary. This is my preferred way to support optional args in a destructuring, as it naturally documents them in the function signature.